utils/mwa_metadb_utils.py

```diff
@@ -130,14 +130,8 @@ def getmeta(service='obs', params=None):
     else:
         data = ''
 
-    if service.strip().lower() in ['obs', 'find', 'con']:
-        service = service.strip().lower()
-    else:
-        logger.error("invalid service name: %s" % service)
-        return
-
     try:
-        result = json.load(urllib.request.urlopen(BASEURL + service + '?' + data))
+        result = json.load(urllib.request.urlopen(BASEURL + servicetype + '/' + service + '?' + data))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Audit url open for permitted schemes. Allowing use of file:/ or custom schemes is often unexpected.](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4557977)